test/routes/admin_stock.test.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const { test } = require('tap')
+const { build } = require('../helper')
+
+// TODO: Update this test
+// - [ ] Reset the stock before testing
+// - [ ] Fix timeout bug
+test('Set stock 0', async (t) => {
+  const app = await build(t)
+
+  const res = await app.inject({
+    method: 'POST',
+    url: '/admin_stock',
+    body: {
+      'machine_id': 'testclient',
+      'item_id': 'd016',
+      'item_stock': 0
+    }
+  })
+  console.log(res.body)
+  t.equal(res.statusCode, 200)

```

Comment:
You should add a stock check here as well. It should just be an app.inject call to /item and you can assert that the d016 stock is 0. I wouldn't bother with this for most tests but here it would work.

Also, you should clean up after yourself and make sure to reset d016 to 100 or whatever value it should default to. You do this with the last test so you can probably not bother, but keep that in mind when writing tests like this